### PR TITLE
Change to use Node.js 0.12

### PR DESCRIPTION
**Pull request by arv**

### CODE REVIEW

**Comment by arv** [comment]
@johnjbarton

**Comment by domenic** [comment]
Why not iojs?

**Comment by arv** [comment]
I wouldn't mind but one step at a time.

**Comment by domenic** [comment]
There's just no real reason to use Node 0.12, the successor to 0.10 should be considered iojs. People either stick to 0.10 or install io.

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "author": "Traceur Authors",
   "license": "Apache License 2.0",
   "engines": {
-    "node": ">=0.10"
+    "node": ">=0.12"
```
Comment (johnjbarton) [inline package.json]: As I understand it we are now compatible with .10 and .12 (and potentially iojs). Is there are reason to force users on to .10?

Comment (pflannery) [inline package.json]: couldn't the engine limitation just be removed?

Comment (johnjbarton) [inline package.json]: I don't want to investigate issues on arbitrary versions if the vast majority of users are on recent versions.

Comment (pflannery) [inline package.json]: I think I got confused by a request I got asking me to remove the engine constraint from a package.json so they could use iojs with my library. But if ```"node": ">=0.12"``` works with iojs then im happy.

Comment (arv) [inline package.json]: I think @johnjbarton convinced me. If we work with Node 0.10 and io.js we should not limit this.

**Pull request closed without merging**